Exercise_A/index.html

```diff
@@ -1,17 +1,46 @@
 <!-- Do not forget to add in the Bootstrap CDN's so that you can utilise the pre-constructed HTML elements -->
 <!DOCTYPE html>
 <html class="no-js">

```

Comment:
Great implementation of this bootstrap website.
Nice work using the bootstrap utilities. 
You could have used a bootstrap class for your list items
list-group-item